layouts/_default/search.html

```diff
@@ -17,33 +17,16 @@
     </header>
     <div class="td-outer">
       <main role="main" class="td-main">
-        <section class="row td-search-result">
+        <section class="row td-search-query">
           <div class="col-12 col-md-4 offset-md-2">
-            <form class="td-sidebar__search d-flex align-items-center">
-              {{ partial "search-input.html" . }}
-            </form>
+            {{- partial "search-form.html" . }}
           </div>
+        </section>
```

Comment:
The search results page also lets you do a new query. I've split that into its own section.